src/System.ServiceModel.Tests.Common/src/OuterLoopTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Newtonsoft.Json;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+
+namespace System.ServiceModel.Tests.Common
+{
+    public abstract class OuterLoopTests : IDisposable
+    {
+        protected string EndpointAddress
+        {
+            get;
+            private set;
+        }
+
+        private string BaseAddress
+        {
+            // Would like to pull this address either from msbuild properties, env vars, or 
+            // configuration passed into xunit.
```

Comment:
Agreed... property of `BaseAddress` and `EndpointAddress` :point_right: :person_frowning: :frowning: :sob: :cry: :crying_cat_face: :-1:

I was bitten really badly previously by having `Constants.SomeAddress` and `Endpoints.SomeAddress` look exactly the same in IntelliSense but point to different places.